docs/plugin_list.md

```diff
@@ -55,18 +55,25 @@ Deploy a contract to multiple chains (1 at a time) with the same address.
 <br>[Documentation](https://github.com/hexdivision/remix-contract-deployer-plugin)
 <br>[Make an issue](https://github.com/hexdivision/remix-contract-deployer-plugin/issues)
 
+**Contract Verification - Etherscan** &nbsp; ![](images/pi-etherscan.png) &nbsp;
+<br>Verify a contract on Etherscan.
+<br>[Profile name](https://github.com/ethereum/remix-plugins-directory/blob/master/plugins/etherscan/profile.json): etherscan
```

Comment:
Etherscan plugin is maintained by Remix. So here is the profile link: https://github.com/ethereum/remix-project/blob/master/apps/etherscan/src/profile.json

Reply:
Documentation: https://remix-ide.readthedocs.io/en/latest/contract_verification.html#etherscan

Reply:
Make an issue: https://github.com/ethereum/remix-project/tree/master/apps/etherscan